redi-connect-front/src/components/organisms/EditableMentoringTopics.tsx

```diff
@@ -156,7 +159,7 @@ const CategoryGroup = ({ id, label, selectedCategories, onChange, formik }: any)
               customOnChange={onChange}
               disabled={
                 !formik.values.isMentor &&
-                selectedCategories.length > 2 &&
+                selectedCategories.length >= MAX_MENTORING_TOPICS_IF_USER_IS_MENTEE &&
```

Comment:
@katamatata your fix was correct - good work.

I added another commit on top of yours to make the code a bit more declarative. We can easily put this "magic number" ([see definition of **magic number** in first bullet item here](https://en.wikipedia.org/wiki/Magic_number_(programming)), i.e. the max number of mentoring topics a mentee can select, into an appropriately named constant. That's better than having the same number in two places, and avoids future problems when we update this code.

Reply:
It was the magic number indeed:)